configs/default/FLMO-STELLARF7.config

```diff
@@ -0,0 +1,134 @@
+# Betaflight / STM32F7X2 (S7X2) 4.4.2 Aug 14 2023 / 20:54:15 (23d066d08) MSP API: 1.45
+
+#define USE_GYRO
+#define USE_GYRO_SPI_MPU6000
+#define USE_ACC
+#define USE_ACC_SPI_MPU6000
+#define USE_ACC_SPI_ICM20602
+#define USE_GYRO_SPI_ICM20602
+#define USE_ACCGYRO_BMI270
+#define USE_BARO
+#define USE_BARO_BMP280
+#define USE_FLASH
+#define USE_FLASH_M25P16
+#define USE_MAX7456
+
+board_name STELLARF7
+manufacturer_id FLMO
+
+# resources
+resource MOTOR 1 C06
+resource MOTOR 2 C07
+resource MOTOR 3 C08
+resource MOTOR 4 C09
+resource PWM 1 B10
+resource PWM 2 B11
+resource PPM 1 B07
+resource SERIAL_TX 1 B06
+resource SERIAL_RX 1 B07
+resource SERIAL_TX 2 A02
+resource SERIAL_RX 2 A03
+resource SERIAL_TX 3 C10
+resource SERIAL_RX 3 C11
+resource SERIAL_TX 4 A00
+resource SERIAL_RX 4 A01
+resource SERIAL_TX 5 C12
+resource SERIAL_RX 5 D02
+resource I2C_SCL 1 B08
+resource I2C_SDA 1 B09
+resource LED 1 B02
+resource SPI_SCK 1 A05
+resource SPI_SCK 2 B13
+resource SPI_SCK 3 B03
+resource SPI_MISO 1 A06
+resource SPI_MISO 2 B14
+resource SPI_MISO 3 B04
+resource SPI_MOSI 1 A07
+resource SPI_MOSI 2 B15
+resource SPI_MOSI 3 B05
+resource ADC_BATT 1 C01
+resource ADC_CURR 1 C00
+resource PINIO 1 C14
+resource PINIO 2 A15
+resource PINIO 3 C15
+resource FLASH_CS 1 C13
+resource OSD_CS 1 B12s
+resource GYRO_EXTI 1 C04
+resource GYRO_CS 1 A04
+resource USB_DETECT 1 A09
+resource BEEPER 1 C03
+
+# timer
+timer B07 AF2
+# pin B07: TIM4 CH2 (AF2)
+timer C06 AF3
+# pin C06: TIM8 CH1 (AF3)
+timer C07 AF3
+# pin C07: TIM8 CH2 (AF3)
+timer C08 AF3
+# pin C08: TIM8 CH3 (AF3)
+timer C09 AF3
+# pin C09: TIM8 CH4 (AF3)
+timer B10 AF1
+# pin B10: TIM2 CH3 (AF1)
+timer B11 AF1
+# pin B11: TIM2 CH4 (AF1)
+
+# dma
+dma ADC 1 1
+# ADC 1: DMA2 Stream 4 Channel 0
+dma pin C06 1
+# pin C06: DMA2 Stream 2 Channel 7
+dma pin C07 1
+# pin C07: DMA2 Stream 3 Channel 7
+dma pin C08 1
+# pin C08: DMA2 Stream 4 Channel 7
+dma pin C09 0
+# pin C09: DMA2 Stream 7 Channel 7
+dma pin B10 0
+# pin B10: DMA1 Stream 1 Channel 3
+dma pin B11 0
+# pin B11: DMA1 Stream 7 Channel 3
+
+# feature
+feature OSD
+feature RX_SERIAL
+feature TELEMETRY
+feature GPS
```

Comment:
Remove cloud build option

```suggestion

```

Reply:
I need to read the documentation, what is now in cloud build and what is in the config file 😁